Jenkinsfile.release

```diff
@@ -64,17 +64,14 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             // only modifies the meta.json
             stage('Replicate AWS AMI') {
                 s3_stream_dir = "${s3_bucket}/prod/streams/${params.STREAM}"
-                // TODO: Once buildprep supports pulling specific builds
-                // operate on the specific build rather than the most
-                // recent build
                 utils.shwrap("""
                 export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
                 coreos-assembler init https://github.com/coreos/fedora-coreos-config
-                coreos-assembler buildprep s3://${s3_stream_dir}/builds
+                coreos-assembler buildprep --build=${params.VERSION} s3://${s3_stream_dir}/builds
                 coreos-assembler aws-replicate --build=${params.VERSION} --log-level=INFO
                 git clone https://github.com/coreos/fedora-coreos-releng-automation /var/tmp/fcos-releng
                 /var/tmp/fcos-releng/coreos-meta-translator/trans.py --workdir .
```

Comment:
And here, we have to pass `--build-id` too.

Reply:
i think I've fixed this now